File: lessons/lesson7.cpp
#include<vector>
#include<fstream>
#include<string>
#include<iostream>


;
int main()
{
    auto s = "xoxoxo\nhello world";
    std::ofstream fw("tests/test.txt");
    fw << s;
    fw.close();

    std::ifstream file("tests/test.txt");
    std::vector<std::string> fr;
    if(file.is_open()){
        std::string line;
        while (getline(file, line)) {
            fr.push_back(line);
        }; file.close();
    }
    std::cout << fr[0] << " " << fr[1] << std::endl;
    for(auto it = fr.begin(); it != fr.end(); ++it)
    {
        auto line = *it;
        std::cout << line << std::endl;
    }
}

File: lessons/lesson7.py


if __name__ == '__main__':
    s = 'xoxoxo\nhello world'

    fw = open('tests/test.txt', 'w')
    fw.write(s)

    fr = open('tests/test.txt', 'r').readlines()
    print(fr[0] + " " + fr[1])

    for line in fr:
        print(line)
